playwright.config.js

```diff
@@ -16,6 +18,8 @@ export default defineConfig({
     baseURL: 'https://www.globalsqa.com/',
     testIdAttribute: 'id'
   },
+      timeout: 60000,  // Global timeout for all tests ***
```

Comment:
There is an extra slash in the comment `// Global timeout for all tests ***`. It should be `// Global timeout for all tests`.